src/main/resources/liquibase/harness/change/changelogs/postgresql/createCompositeType.json

```diff
@@ -7,12 +7,12 @@
         "changes": [
```

Comment:
i already fixed files for createCompositeType in scope of renameCompositeTypeAttribute PR, including json structure in expectedJson - better to revert changes in createCompositeType related files. 
To avoid conflicts when pulling latest main